includes/wf_crm_webform_postprocess.inc

```diff
@@ -764,6 +760,9 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
           // start array index at 1
           $existing = array_merge(array(array()), $result['values']);
         }
+        // Check which location_type_id is to be set as is_primary=1;
+        $is_primary_location_type = isset($contact[$location][1]['location_type_id']) ? $contact[$location][1]['location_type_id'] : null;
```

Comment:
This could be written more concisely as

```suggestion
        $is_primary_location_type = $contact[$location][1]['location_type_id'] ?? NULL;
```

Reply:
@colemanw I made this change and it works fine. 

But just to note, the is only available on PHP 7.x and I guess its ok for 7.x-5.x branch.